### DISCUSSION TITLE
Excited for Gatsby Tutorial Part 6

### DISCUSSION THREAD

**Discussion by rohitjpsingh-zz** [gatsbyjs/gatsby#32529] [Help] [answered]
I've read gatsby tutorial. it is very easy and transparent for beginners. now I completed all part 5 and excited waiting for part 6. I think it launch date is today.
May I know when It'll be released.
Thank you @gatsbyjs

**Comment by LekoArts** [comment, marked as answer]
Hi!

Great to hear that you enjoy it :) The PR adding part 6 will most likely be merged in about 6~9 hours (before the livestream later this day: https://www.twitch.tv/gatsbyjs/schedule)
